spec/models/podcast_episode_spec.rb

```diff
@@ -9,11 +9,6 @@
   it { is_expected.to validate_presence_of(:guid) }
 
   describe "validations" do
-    # Couldn't use shoulda matchers for these tests because:
```

Comment:
This comment can save some time in case anyone wants to refactor to `validates_uniqueness_of`

Reply:
makes sense, i'll put them back :)